src/Orchard.Web/Modules/Orchard.ContentPreview/Views/Preview/Index.cshtml

```diff
@@ -0,0 +1,156 @@
+@using Orchard.UI.Resources
+
+@{
+    var previewId = ViewContext.RouteData.Values["id"];
+    Script.Require("jQuery").AtHead();
+    Script.Include("js.cookie.js");
```

Comment:
Ez footban lesz, jQuery headben. Biztos így akartad?

Reply:
AtHeadet adtam a másiknak is, ennyinek szerintem fölösleges manifest dependencyvel.